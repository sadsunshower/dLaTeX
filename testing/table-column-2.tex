\documentclass{article}

\begin{document}

Test: table column mismatch

\begin{tabular}{lll}
    a & b & c & d \\
    a & b & c \\
\end{tabular}

\end{document}