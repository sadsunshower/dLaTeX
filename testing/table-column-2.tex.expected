This is dLaTeX version 0.1
{"messages": [{"severity": "error", "message": "This row contains extra column(s). For typing a literal & use \\&, otherwise check your options for the environment.", "line": "    a & b & c & d \\\\", "num": 8, "marker": [14]}], "error": true}
